modules/gapi/src/backends/common/gbackend.hpp

```diff
@@ -58,6 +58,35 @@ namespace gimpl {
     struct Data;
     struct RcDesc;
 
+    struct RMatMediaBGRAdapter final: public cv::RMat::Adapter
+    {
+        RMatMediaBGRAdapter(cv::MediaFrame frame) : m_frame(frame) { };
```

Comment:
const reference

Reply:
That class is reference-counted anyway, but yes it may make sense.

I'd say this constructor must be `explicit` too.

Reply:
Fixed